### ISSUE TITLE
vp9 encoder missing

### ISSUE THREAD

**Issue by marcokuchla** [skvark/opencv-python#86]
### Expected behaviour

VP9 encoder available to use because it is supposed to be compiled inside manylinux custom image:
`... cd ~/ffmpeg_sources && git clone --depth 1 https://chromium.googlesource.com/webm/libvpx.git && cd libvpx && ./configure --prefix="$HOME/ffmpeg_build" --disable-examples --disable-unit-tests --enable-vp9-highbitdepth --as=yasm --enable-pic && make -j4 && make install ... --enable-libvpx ...`

### Actual behaviour

Encoder is missing.

### Steps to reproduce

I've installed opencv-contrib-python (3.4.0.12) inside a python:3 Docker image. Then the following commands:
```
>>> import cv2
>>> writer = cv2.VideoWriter()
>>> writer.open('video.webm', cv2.VideoWriter_fourcc(*'VP90'), 30, (100,100))
OpenCV: FFMPEG: tag 0x30395056/'VP90' is not supported with codec id 167 and format 'webm / WebM'
Could not find encoder for codec id 167: Encoder not found
False
```

**Comment by skvark** [comment]
Yes, it seems that vp9 is missing. Similar issue: https://www.centos.org/forums/viewtopic.php?t=44422

**Comment by skvark** [comment]
I'll try to reorder the FFmpeg build flags, it seems that the order matters.

**Comment by skvark** [comment]
Solved it. ``--enable-shared`` was missing from ``libvpx`` build flags. FFmpeg build detected ``libvpx`` but did not actually use it. Should be ok now. I'm currently building ``x86_64`` image. Both of the fixed images should be up soon and the Linux builds should start passing again.
